src/gui/tray/ActivityItemActions.qml

```diff
@@ -46,10 +46,11 @@ RowLayout {
             imageSource: model.modelData.imageSource ? model.modelData.imageSource + root.adjustedHeaderColor : ""
             imageSourceHover: model.modelData.imageSourceHovered ? model.modelData.imageSourceHovered + UserModel.currentUser.headerTextColor : ""
 
-            textColor: imageSource !== "" ? root.adjustedHeaderColor : Style.ncTextColor
-            textColorHovered: imageSource !== "" ? UserModel.currentUser.headerTextColor : Style.ncTextColor
+            textColor: imageSource !== "" || primary ? root.adjustedHeaderColor : Style.ncTextColor
+            textColorHovered: imageSource !== "" || primary ? UserModel.currentUser.headerTextColor : Style.ncTextColor
```

Comment:
Throughout `ActivityItemActions` we are now basically always using the `CustomButton` for a situation where we have a primary button (before we would use `primary` to ensure we bolded the first button in the `CustomTextButton`)

Now that we always set things with `imageSource` and `primary` together, can we not just integrate `imageSource !== ""` into the `primary` property?